Tests/SwiftIfConfigTest/ActiveRegionTests.swift

```diff
@@ -98,3 +101,45 @@ public class ActiveRegionTests: XCTestCase {
     )
   }
 }
+
+/// Assert that the various marked positions in the source code have the
+/// expected active states.
+func assertActiveCode(
```

Comment:
This function could now be `fileprivate` because it’s only being used in this file.